bot/exts/utils/thread_bumper.py

```diff
@@ -24,6 +24,23 @@ class ThreadBumper(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    async def thread_exists_in_site(self, thread_id: int) -> bool:
+        """Return whether the given thread_id exists in the site api's bump list."""
+        # If the thread exists, site returns a 204 with no content.
+        # Due to this, `api_client.request()` cannot be used, as it always attempt to decode the response as json.
```

Comment:
`attempt` -> `attempts`

Reply:
I fixed both this and the need to await the async function (who knew this was needed 🤦) with a forcepush to the most recent commit.